compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/platform/JvmActuals.jvm.kt

```diff
@@ -16,7 +16,14 @@
 
 package androidx.compose.ui.platform
 
-internal actual typealias AtomicInt = java.util.concurrent.atomic.AtomicInteger
+internal class AtomicIntWrapper(value_: Int) {
```

Comment:
because in expect constructor  the parameter name is 'value_' - doesn't match the name in `java.util.concurrent.atomic.AtomicInteger` (value)